### PR TITLE
staging and publish directives for asf

### PR DESCRIPTION
**Pull request by DaanHoogland**

### CODE REVIEW

**Comment by Humbedooh** [comment]
For this to apply to the asf-site branch (and thus publish it), it will also need to be present in that branch after this PR is merged.

**Comment by DaanHoogland** [comment]
> For this to apply to the asf-site branch (and thus publish it), it will also need to be present in that branch after this PR is merged.

yes, that is part of the procedure described in the README. (i just noticed an errror in that large chunk of code i submitted here, so I am amending a bit ;)

File: .asf.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Staging and publishing profile for yourproject-website.git:
+staging:
+  profile: ~
+  whoami:  master
```
Comment (DaanHoogland) [inline .asf.yaml:4]:
```suggestion
  whoami:  main
```

**Comment by PaulAngus** [comment]
🤷🏻‍♂️ per the instructions, looks good to me.  We can quickly revert if it goes pear-shaped.

**Comment by svenvogel** [comment]
i see the same. looks good from definition :thinking:

**Comment by DaanHoogland** [comment]
I won't make a habbit of it but as no one is merging this I will. I won't merge it in asf-site as this will happen automatically the first time something needs publishing.

**Pull request merged**